config/overlays/base/kustomization.yaml

```diff
@@ -1,7 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Any reason to change the dir name for this file?

Reply:
Because `rhmp`  is another overlay and it is also one of the `olm` bundle types, I agree that `base` dir name is not perfect but it also cannot be called `olm`, suggestions for the other name are more than welcome.